README.md

```diff
@@ -1,2 +1,28 @@
 # tetrio_flatpak
 A flatpak build of tetr.io
+
+# Build instructions:
+
+### Install flatpak-builder
+
+Debian:
+
+    sudo apt install flatpak-builder
+
+Arch:
+
+    sudo pacman -S flatpak-builder
+
+Fedora: 
+
+    sudo dnf install flatpak-builder
+
```

Comment:
This can be simplified by just doing `flatpak install org.flatpak.Builder`, will work on every distro that supports flatpak.